src/main/java/org/opensearch/knn/indices/ModelDao.java

```diff
@@ -334,18 +334,18 @@ public ModelMetadata getMetadata(String modelId) {
             IndexMetadata indexMetadata = clusterService.state().metadata().index(MODEL_INDEX_NAME);
 
             if (indexMetadata == null) {
-                throw new RuntimeException("Model index's metadata does not exist");
```

Comment:
shall we log this exception message so that we know why metadata is null?

Reply:
debug is good since it is not an information to user.

Reply:
I see. Updated.